src/qibolab/transpilers/general_connectivity.py

```diff
@@ -230,7 +230,7 @@ def subgraph_init(self):
         self._mapping = result.mapping
 
     def greedy_init(self):
-        """initialize the circuit with greedy algorithm let a maximum number of 2-qubit
+        """Initialize the circuit with greedy algorithm let a maximum number of 2-qubit
```

Comment:
This commit has definitely a wrong message...

Reply:
I didn't know how to run tests again ahhahahaha